ipam/store.go

```diff
@@ -39,6 +39,12 @@ func (a *Allocator) Value() []byte {
 	return b
 }
 
+// SetValue unmarshalls the data from the KV store.
+func (a *Allocator) SetValue(value []byte) error {
+	a.subnets = byteArrayToSubnets(value)
```

Comment:
`byteArrayToSubnets()` doesn't return an error.  Are you asking me to change it so that it does?